### ISSUE TITLE
MGHPCC 2023 Shutdown Website Banner takedown

### ISSUE THREAD

**Issue by joachimweyl** [CCI-MOC/ops-issues#930]
- [ ] Request takedown of the [website banner](https://github.com/CCI-MOC/ops-issues/issues/911) once systems are back up (June 9th-12th be prepared)

**Comment by joachimweyl** [comment]
@taramoran are we ready to take down the shutdown banner when the engineers give the go-ahead?

**Comment by taramoran** [comment]
@joachimweyl yes! let me know when, or if it's over the weekend and can't wait until Monday you also have editing access at massopen.cloud/wp-admin and should be able to just remove that section

**Comment by taramoran** [comment]
@joachimweyl done! will be pushed out in the next few hours
